src/pybind/mgr/nfs/ganesha_conf.py

```diff
@@ -243,15 +249,15 @@ def to_fsal_block(self) -> RawBlock:
     @classmethod
     def from_dict(cls, fsal_dict: Dict[str, Any]) -> 'CephFSFSAL':
         return cls(fsal_dict['name'],
+                   fsal_dict['cmount_path'],
```

Comment:
I don't expect this function to work correctly. The 2nd argument to the __init__ function is (still) user_id. I suggest changing this `from_dict` call to use keyword arguments and avoid having to reshuffle argument order.

Reply:
I tried testing the current patches. I hit the following error. I suspect it may be related to this change:
```
[nfs ERROR nfs.export] Failed to create /foo export for nfs1
Traceback (most recent call last):
  File "/usr/share/ceph/mgr/nfs/export.py", line 401, in create_export
    return self.create_cephfs_export(**kwargs)
  File "/usr/share/ceph/mgr/nfs/export.py", line 671, in create_cephfs_export
    export = self.create_export_from_dict(
  File "/usr/share/ceph/mgr/nfs/export.py", line 642, in create_export_from_dict
    export.validate(self.mgr)
  File "/usr/share/ceph/mgr/nfs/ganesha_conf.py", line 511, in validate
    raise FSNotFound(fs.fs_name)
nfs.exception.FSNotFound: filesystem None not found
```